src/netspresso_trainer/pipelines/task_processors/classification.py

```diff
@@ -26,7 +26,7 @@ class ClassificationProcessor(BaseTaskProcessor):
     def __init__(self, conf, postprocessor, devices, **kwargs):
         super(ClassificationProcessor, self).__init__(conf, postprocessor, devices, **kwargs)
 
-    def train_step(self, train_model, batch, optimizer, loss_factory, metric_factory):
+    def train_step(self, train_model, batch, optimizer, loss_factory, metric_factory, model_max_norm: Optional[float]=None):
```

Comment:
I think better to have `model_max_norm` as member of processor task, not parameter of `train_step`.

Reply:
Thank you for the review. I agree with your opinion. It would be better to have it as a member of the task instance for maintaining the code base. I'll update it and commit soon😊